presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -2285,6 +2286,29 @@ public void testCreateExternalTableTextFileFieldSeparatorEscape()
         deleteRecursively(tempDir.toPath(), ALLOW_INSECURE);
     }
 
+    @Test
+    public void testCreateExternalTableWithCTASWritesDisabled()
+            throws IOException
+    {
+        File tempDir = createTempDir();
+        File dataFile = new File(tempDir, "test.txt");
+
+        @Language("SQL") String createTableSql = format("" +
+                        "CREATE TABLE %s.%s.test_create_external " +
+                        "WITH (\n" +
+                        "   external_location = '%s',\n" +
+                        "   format = 'TEXTFILE',\n" +
+                        "   textfile_field_separator = U&'\\0001'\n" +
```

Comment:
`format` and `textfile_field_separator` are redundant from this test's point of view, so you can wrap this as:
```
"WITH (external_location = '%s') AS " +
```